src/main/java/io/swagger/codegen/languages/kotlin/KotlinClientCodegen.java

```diff
@@ -0,0 +1,127 @@
+package io.swagger.codegen.languages.kotlin;
+
+import io.swagger.codegen.CliOption;
+import io.swagger.codegen.CodegenConstants;
+import io.swagger.codegen.CodegenType;
+import io.swagger.codegen.SupportingFile;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class KotlinClientCodegen extends AbstractKotlinCodegen {
+
+    public static final String DATE_LIBRARY = "dateLibrary";
+    protected CodegenConstants.ENUM_PROPERTY_NAMING_TYPE enumPropertyNaming = CodegenConstants.ENUM_PROPERTY_NAMING_TYPE.camelCase;
```

Comment:
`enumPropertyNaming` is already declared in `AbstractKotlinCodegen`. You just need to change the value (in the constructor for example) in the child class.